plugins/notifications/src/components/NotificationsTable/NotificationsTable.tsx

```diff
@@ -223,20 +223,22 @@ export const NotificationsTable = ({
               <Grid item xs={11}>
                 <Box>
                   <Typography variant="subtitle2">
-                    {notification.payload.link ? (
-                      <Link
-                        to={notification.payload.link}
-                        onClick={() => {
-                          if (markAsReadOnLinkOpen && !notification.read) {
-                            onSwitchReadStatus([notification.id], true);
-                          }
-                        }}
-                      >
-                        {notification.payload.title}
-                      </Link>
-                    ) : (
-                      notification.payload.title
-                    )}
+                    <Box sx={{ fontWeight: 'bold' }}>
```

Comment:
is there a Typography variant that you can use instead? thinking h5 or h6 might fit

Reply:
Those h5 or h6 are still to big, distracting.
Trying `subtitle1`. Please see the later screenshot. Wdyt?